nodeJS/introduction_to_nodeJS/debugging_node.md

```diff
@@ -12,9 +12,9 @@ By the end of this lesson, you should be able to do the following:
 
 <div class="lesson-content__panel" markdown="1">
 
+- Watch [this video tutorial](https://www.youtube.com/watch?v=2oFKNL7vYV8&ab_channel=VisualStudioCode) to see the process in action.
```

Comment:
```suggestion
- Watch [this video tutorial](https://www.youtube.com/watch?v=2oFKNL7vYV8&ab_channel=VisualStudioCode) to see the Node.js debugging in VS Code process in action.
```

Looks good to me and thanks for taking this on. However, I think making this a bit more descriptive might be helpful. What do you think @roy9495 ?

Reply:
Yes, This description will be more specific. I will commit this.

Reply:
You can either click the `commit suggestion` button below my review, or make a commit yourself and push it :)

Reply:
Sorry for the delay .

Reply:
no need to apologise at all! Thanks for doing this :)